icons/pac-man.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m12 12 7.4 6.7a10 10 0 1 1 0-13.4Z" />
+  <path d="M18 12h.01" />
+  <path d="M22 12h.01" />
+</svg>
```

Comment:
Not public domain :-1:

Reply:
?

Reply:
The interlectual property rights are held by namco I think.

Reply:
> The interlectual property rights are held by namco I think.

I see. Still? Does that not then also apply to the previous `ghost`?

Reply:
> I see. Still?

Well, it's hardly been [_that_ long](https://en.m.wikipedia.org/wiki/Pac-Man_99) since the last pacman game was released.

> Does that not then also apply to the previous `ghost`?

I don't think so, at least not as long as we don't call it a pacman ghost. ;)

Reply:
I was going off `pac-man` being in the previous `ghost.json`… So call this something else, (`chomp` or something), or remove? @jguddas @karsa-mistmere